Shared/sdk/SString.h

```diff
@@ -30,6 +30,7 @@ class SString : public std::string
 
     SString(const char* szText) : std::string(szText ? szText : "") {}
 
+    template <int=0>
```

Comment:
What is the purpose of it?

Reply:
This template hack will force C++ to properly recognize varargs and string constructors

Reply:
Please, add a comment describing it.